src/components/accordion-with-image/accordionInfo.js

```diff
@@ -0,0 +1,25 @@
+import MapLogo from '~/img/guest-home-page/map.svg'

```

Comment:
All images should be in the assets/images folder.

Reply:
++